pom.xml

```diff
@@ -147,12 +147,23 @@
                 <version>1.5.0-SNAPSHOT</version>
             </dependency>
             <dependency>
+                <groupId>com.netflix.hystrix</groupId>
+                <artifactId>hystrix-core</artifactId>
+                <version>1.3.16</version>
+            </dependency>
+            <dependency>
                 <groupId>com.redhat.lightblue</groupId>
                 <artifactId>lightblue-core-test</artifactId>
                 <version>1.5.0-SNAPSHOT</version>
                 <scope>test</scope>
             </dependency>
             <dependency>
+                <groupId>com.redhat.lightblue.mongo</groupId>
+                <artifactId>lightblue-mongo-test</artifactId>
```

Comment:
An undocumented pattern we've been following is to only put dependencies outside of the repo in dependencyManagement.  In my mind it's defining an artifact that doesn't exist yet doing this, but it technically works.